cmd/pebble/cmd_enter.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/canonical/pebble/internal/logger"
+	"github.com/jessevdk/go-flags"
+)
+
+const shortEnterHelp = "Run subcommand under a container environment"
+const longEnterHelp = `
+The enter command facilitates the use of Pebble as an entrypoint for containers.
+When used without a subcommand it mimics the behavior of the run command
+alone, while if used with a subcommand it runs that subcommand in the most
+appropriate environment taking into account its purpose.
+
+These subcommands are currently supported:
+
+  help      (1)(2)
+  version   (1)(2)
+  plan      (1)(2)
+  services  (1)(2)
+  ls        (1)(2)
+  exec      (2)
+  start     (3)
+  stop      (3)
+
+(1) Services are not started.
+(2) No logs on stdout unless -v is used.
+(3) Services continue running after the subcommand succeeds.
+`
+
+type enterFlags int
+
+const (
+	enterSilenceLogging enterFlags = 1 << iota
+	enterNoServiceManager
+	enterKeepServiceManager
+	enterRequireServiceAutostart
+	enterProhibitServiceAutostart
+)
+
+type cmdEnter struct {
+	clientMixin
+	sharedRunEnterOpts
+	Run        bool `long:"run"`
+	Positional struct {
+		Cmd []string `positional-arg-name:"<subcommand>"`
+	} `positional-args:"yes"`
+}
+
+func init() {
+	optsHelp := map[string]string{
+		"run": "Start \"autostart\" services before executing subcommand",
```

Comment:
s/Start "autostart" services/Start default services/

This is the same terminology used in the autostart command itself.